api/collections/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url
+from api.collections import views
+
+urlpatterns = [
+    # Examples:
+    # url(r'^$', 'api.views.home', name='home'),
+    # url(r'^blog/', include('blog.urls')),
+    url(r'^$', views.CollectionList.as_view(), name='collection-list'),
+    url(r'^dashboard/$', views.DashboardDetail.as_view(), name='dashboard-detail'),
```

Comment:
If it's a lot more complicated than the extra view, then go ahead and keep it, but you might be able to dump a bunch of lines of code now.